packages/app/src/commands/modernize/checkRequirements.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs')
+const path = require('path')
+const { reporter } = require('@dhis2/cli-helpers-engine')
+
+/**
+ * @param {Object} args
+ * @param {string} args.inDir
+ * @param {string} args.outDir
+ * @param {string} args.primaryLanguage
+ * @param {string[]} args.translationFiles
+ * @return {void}
+ */
+const checkRequirements = ({
+    inDir,
+    outDir,
+    primaryLanguage,
+    translationFiles,
+}) => {
+    const inDirExists = fs.existsSync(inDir)
```

Comment:
These checks should probably also confirm that `inDir` is actually a directory, not a file.  You can do that with `fs.existsSync(inDir) && fs.statSync(inDir).isDirectory()` (see [here](https://github.com/dhis2/cli/pull/126/files#diff-f0dae67115941bb5d80b603159a41f59R111-R115))